nilearn/plotting/html_surface.py

```diff
@@ -21,20 +21,39 @@ class SurfaceView(HTMLDocument):
 
 
 def _get_vertexcolor(surf_map, cmap, norm,
-                     absolute_threshold=None, bg_map=None):
+                     absolute_threshold=None, bg_map=None,
+                     bg_on_data=None, scale_bg_map=None, darkness=None):
     vertexcolor = cmap(norm(surf_map).data)
-    if absolute_threshold is None:
-        return to_color_strings(vertexcolor)
+
     if bg_map is None:
         bg_map = np.ones(len(surf_map)) * .5
         bg_vmin, bg_vmax = 0, 1
     else:
         bg_map = surface.load_surf_data(bg_map)
+
+    # scale background map if need be
+    if scale_bg_map:
```

Comment:
doesn't really matter but you only need this when bg_map is not None?

Reply:
Yes, I chose to keep a "flat" coding structure here, thinking it'd be easier to maintain.
The previous `if/else` block instantiates `bg_map`, this one instantiates `bg_data`, etc, and I thought it be clearer this way.